.github/workflows/coverage.yml

```diff
@@ -0,0 +1,35 @@
+# editorconfig-checker-disable-file
+---
+name: coverage
+
+on:
+    push:
+        branches:
+        - main
+        - devel
+    pull_request:
+        branches:
+        - main
+        - devel
+
+jobs:
+    test:
+        name: coverage
```

Comment:
awesome work, can you point me to some docs on this?

Reply:
https://github.com/xd009642/tarpaulin#github-actions added as a comment to the job as well